src/pages/subreddit/Subreddit.js

```diff
@@ -0,0 +1,75 @@
+import React, {useEffect, useState} from 'react';
+import {Link, useParams} from "react-router-dom";
+import axios from "axios";
+import './Subreddit.css'
+import Header from "../../components/Header";
+
+function SubredditPage() {
+
+
+    const [redditDetails, setRedditDetails] = useState({});
+    const {id} = useParams();
+
+    useEffect(() => {
+        async function getDetails() {
+
+            try {
+
+                const response = await axios.get(`https://www.reddit.com/r/${id}/about.json`)
+                console.log(response.data.data)
+                setRedditDetails(response.data.data)
+
+            } catch (e) {
+                console.log(e)
+            }
+
+        }
+
+
+        getDetails()
+
+    }, [id])
+
+
+    return (
+
+
+        <>
+
+            <Header
+
+                nameOfListItem1="hottest posts"
+                nameOfListItem2="reddit"
+                nameOfListItem3="memes"
+                pageTitle={id}
+                pageSubtitle="Subreddit Specifications"
+            />
+
+            <main className="subreddit-specs-page">
+                <section className="outer-container-specs">
+                    <div className="inner-container-specs">
+                        {Object.keys(redditDetails).length > 0 && (
```

Comment:
Je check wordt hier dan ook iets simpeler
```suggestion
                        {redditDetails && (
```